app/api/routes/recommendation_events.py

```
@@ -0,0 +1,126 @@
+        page=page,
+        page_size=page_size,
+        items=formatted_items,
+    )
+
+
+@router.get("/users/{user_id}", response_model=EventListResponse)
+def get_user_events(
+    user_id: int,
+    page: int = Query(1, ge=1, description="페이지 번호"),
+    page_size: int = Query(50, ge=1, le=100, description="페이지 크기"),
+    db_mongo: Database = Depends(get_mongo_db),
```

Ruff (0.14.6)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)